consensus/scp/core/src/quorum_set.rs

```diff
@@ -430,9 +429,26 @@ impl<ID: GenericNodeId + AsRef<ResponderId>> From<&QuorumSet<ID>> for QuorumSet<
 mod quorum_set_tests {
     use super::*;
     use crate::{core_types::*, msg::*, predicates::*, test_utils::test_node_id};
-    use mc_common::ResponderId;
-    use std::collections::hash_map::DefaultHasher;
+    use core::hash::{BuildHasher, Hash, Hasher};
+    use mc_common::HasherBuilder;
```

Comment:
Again nit - why is this being used?

Reply:
Because `mc-consensus-scp-core` is `no_std` (because `mc-transaction-core` is `no_std`), and this is a compatible hasher builder within easy reach.